Core/DolphinVM/ist.h

```diff
@@ -75,15 +75,19 @@
 #pragma warning(pop)
 #include <functional>
 
+typedef _Return_type_success_(return >= 0) int32_t NTSTATUS;
+#define NTSTATUS_DEFINED
+#define _NTDEF_
+
 #include "Environ.h"
 
-typedef signed char		SBYTE;
-typedef short			SWORD;
-typedef long			SDWORD;
+typedef int8_t		SBYTE;
```

Comment:
Eventually these will be removed, but at present they are still used elsewhere in the VM